heidrun/smithsonian.rb

```diff
@@ -0,0 +1,446 @@
+# coding: utf-8
+
+CREATOR_LABEL_VALUES = [
+  'Architect',
+  'Artist',
+  'Artists/Makers',
+  'Attributed to',
+  'Author',
+  'Cabinet Maker',
+  'Ceramist',
+  'Circle of',
+  'Co-Designer',
+  'Creator',
+  'Decorator',
+  'Designer',
+  'Draftsman',
+  'Editor',
+  'Embroiderer',
+  'Engraver',
+  'Etcher',
+  'Executor',
+  'Follower of',
+  'Graphic Designer',
+  'Instrumentiste',
+  'Inventor',
+  'Landscape Architect',
+  'Landscape Designer',
+  'Maker',
+  'Model Maker/maker',
+  'Modeler',
+  'Painter',
+  'Photographer',
+  'Possible attribution',
+  'Possibly',
+  'Possibly by',
+  'Print Maker',
+  'Printmaker',
+  'Probably',
+  'School of',
+  'Sculptor',
+  'Studio of',
+  'Workshop of',
+  'Weaver',
+  'Writer',
+  'animator',
+  'architect',
+  'artist',
+  'artist.',
+  'artist?',
+  'artist attribution',
+  'author',
+  'author.',
+  'author?',
+  'authors?',
+  'caricaturist',
+  'cinematographer',
+  'composer',
+  'composer, lyricist',
+  'composer; lyrcist',
+  'composer; lyricist',
+  'composer; performer',
+  'composer; recording artist',
+  'composer?',
+  'creator',
+  'creators',
+  'designer',
+  'developer',
+  'director',
+  'editor',
+  'engraver',
+  'ethnographer',
+  'fabricator',
+  'filmmaker',
+  'filmmaker, anthropologist',
+  'garden designer',
+  'graphic artist',
+  'illustrator',
+  'inventor',
+  'landscape Architect',
+  'landscape architect',
+  'landscape architect, photographer',
+  'landscape designer',
+  'lantern slide maker',
+  'lithographer',
+  'lyicist',
+  'lyicrist',
+  'lyricist',
+  'lyricist; composer',
+  'maker',
+  'maker (possibly)',
+  'maker or owner',
+  'maker; inventor',
+  'original artist',
+  'performer',
+  'performer; composer; lyricist',
+  'performer; recording artist',
+  'performers',
+  'performing artist; recipient',
+  'performing artist; user',
+  'photgrapher',
+  'photograher',
+  'photographer',
+  'photographer and copyright claimant',
+  'photographer and/or colorist',
+  'photographer or collector',
+  'photographer?',
+  'photographerl',
+  'photographerphotographer',
+  'photographers',
+  'photographers?',
+  'photographer}',
+  'photographic firm',
+  'photogrpaher',
+  'playwright',
+  'poet',
+  'possible maker',
+  'printer',
+  'printmaker',
+  'producer',
+  'recordig artist',
+  'recording artist',
+  'recording artist; composer',
+  'recordist',
+  'recordng artist',
+  'sculptor',
+  'shipbuilder',
+  'shipbuilders',
+  'shipping firm',
+  'weaver',
+  'weaver or owner'
+]
+
+def creator?(value)
+  CREATOR_LABEL_VALUES.include?(value)
+end
+
+clean_language = lambda do |i|
+  i.value.gsub(/ languages?/i, '')
```

Comment:
Just to be clear; this is meant to match exactly two strings: `" language"` and ` "languages"`. Correct?